apps/src/blockly/addons/contextMenuAndShortcut.js

```diff
@@ -1,65 +1,16 @@
 import GoogleBlockly from 'blockly/core';
+import {CrossTabCopyPaste} from '@blockly/plugin-cross-tab-copy-paste';
 
-/**
- * Adds a copy command to the block context menu. After switching to v7,
- * we can replace this with:
- * https://github.com/google/blockly-samples/tree/master/plugins/cross-tab-copy-paste
- */
-const registerBlockCopyToStorage = function() {
-  const copyToStorageOption = {
-    displayText: function() {
-      return 'Copy';
-    },
-    preconditionFn: function(scope) {
-      if (scope.block.isDeletable() && scope.block.isMovable()) {
-        return 'enabled';
-      } else {
-        return 'disabled';
-      }
-    },
-    callback: function(scope) {
-      const copyData = Blockly.Xml.domToPrettyText(
-        Blockly.Xml.blockToDom(scope.block)
-      );
-      localStorage.setItem('blocklyStash', copyData);
-    },
-    scopeType: GoogleBlockly.ContextMenuRegistry.ScopeType.BLOCK,
-    id: 'blockCopyToStorage',
-    weight: 0
-  };
-  GoogleBlockly.ContextMenuRegistry.registry.register(copyToStorageOption);
+const options = {
+  contextMenu: true,
+  shortcut: true
```

Comment:
Is this line the only thing here that has to do with shortcuts? Rather than renaming this file, I wonder if it would be cleaner to import and initialize the plugin elsewhere (maybe right in the wrapper), and leave this file exclusively for _our_ custom context menu options.

Reply:
Sure - that sounds good to me!